handbook/company/README.md

```diff
@@ -115,6 +115,12 @@ To provide clarity about decision-making, responsibility, and resources, everyon
 ## Product groups
 Above and beyond the organizational chart, Fleet organizes cross-functional groups focused on particular business goals.  This helps product development teams move more quickly by eliminating roundtrips spent waiting for feedback and answers from people in other departments. Product groups include a designer, a product quality lead, developers, a product manager, and an engineering manager.  For more information, check out the ["Product groups"](./development-groups.md) page.
 
+## Our Sales
+Instead of having sales people 'spray and pray' messages, we take a different approach.
+We have engineers reach out to organizations using Fleet and ask 'how's it going'.
+There's multiple versions of Fleet - we want to help you pick the right one.
+(https://github.com/fleetdm/confidential/blob/main/free-for-all/cold-outbound-strategy.md)
```

Comment:
Thanks for the quick turnaround Ringo.

Could you instead add this to the bottom of https://fleetdm.com/handbook/company/why-this-way, pattern-matching off of how the other headings are laid out?